Nodejs/Product/Nodejs/Debugger/BinaryDeserializer.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.IO;
+using TypeScriptSourceMapReader;
+
+namespace Microsoft.NodejsTools.Debugger {

```

Comment:
As we spoke today, this, as well as the other newly added files, were imported as is. It's a valid point, but we should change them in the original source location where they'd be shared from.